contributing.md

```diff
@@ -10,6 +10,7 @@ Ensure your pull request adheres to the following guideline
 
 > Hacktoberfest's value #2 **Quantity is fun, quality is key**
 
+> Be wary of this before posting **Spammy PRs can be labeled as “invalid.”**
```

Comment:
Could you add more text here? An example? Maybe "Just adding oneself to contributing.md will probably marked as invalid."

Also be sure to use https://github.com/DavidAnson/markdownlint to check proper Markdown format. There is a vs code plugin: https://github.com/DavidAnson/vscode-markdownlint